src/Tooltip.js

```diff
@@ -110,10 +110,12 @@ const defaultProps = {
 }
 
 const Tooltip = ({ children, ...props }) => {
+  const { align, ..._props } = props
+
   return (
     <div style={{ position: 'relative', zIndex: props.zIndex }}>
-      <TooltipContent p={2} mb={3} mt={2} {...props}>
-        {children}
+      <TooltipContent p={2} mb={3} mt={2} {..._props}>
+        <Text align={'center'}>{children}</Text>
```

Comment:
Surprised that Prettier didn't pick this up
```suggestion
        <Text align="center">{children}</Text>
```